presto-spi/src/main/java/io/prestosql/spi/block/VariableWidthBlockBuilder.java

```diff
@@ -47,10 +47,10 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(VariableWidthBlockBuilder.class).instanceSize();
 
-    private BlockBuilderStatus blockBuilderStatus;
+    private final BlockBuilderStatus blockBuilderStatus;
 
     private boolean initialized;
-    private int initialEntryCount;
+    private final int initialEntryCount;
```

Comment:
Should we also group initial, immutable state before mutable state?

```
private final BlockBuilderStatus blockBuilderStatus;
private final int initialEntryCount;

private boolean initialized;
...
```

Reply:
That sounds good for a followup PR. It would affect classes like `Int128ArrayBlockBuilder` which already have these fields as final.